src/MongoDB.Driver/IndexKeysDefinitionBuilder.cs

```diff
@@ -44,6 +44,21 @@ public static IndexKeysDefinition<TDocument> Ascending<TDocument>(this IndexKeys
             return builder.Combine(keys, builder.Ascending(field));
         }
 
+        /// <summary>
+        /// Combines an existing index keys definition with an ascending wildcard index key definition.
+        /// </summary>
+        /// <typeparam name="TDocument">The type of the document.</typeparam>
+        /// <param name="keys">The keys.</param>
+        /// <param name="field">The wildcard key.</param>
+        /// <returns>
+        /// A combined index keys definition.
+        /// </returns>
+        public static IndexKeysDefinition<TDocument> AscendingWithWildcard<TDocument>(this IndexKeysDefinition<TDocument> keys, FieldDefinition<TDocument> field = null)
```

Comment:
I don't think wild card indexes have an ascending/descending direction.

We need a different name for this method.

Reply:
Yes, my mistake. Renamed to `Wildcard`. Another possible name can be `WildcardKey`.